src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -455,15 +459,20 @@ public static bool IsHotReloadable()
     }
 
     /// <summary>
-    /// Helper function to check if multiple create option is enabled.
+    /// Helper method to check if multiple create option is supported and enabled.
+    /// 
+    /// Returns true when
+    /// 1. Multiple create operation is supported by the database type and
+    /// 2. Multiple create operation is enabled in the runtime config.
+    /// 
     /// </summary>
-    public bool IsMultipleCreateOptionEnabled()
+    public bool IsMultipleCreateOperationSupportedAndEnabled()
```

Comment:
Good catch..

Agree that the caller of the function only needs to worry about whether the result is `true`/`false`. It does not matter if it is false because of underlying db type not supporting the operation or the operation not being enabled in the config file.

I've updated all the function and variable names to `IsMultipleCreateEnabled`